src/librbd/LibrbdWriteback.cc

```diff
@@ -163,6 +168,8 @@ namespace librbd {
 				 object_no, off, objectx, object_overlap,
 				 bl, snapc, snap_id,
 				 req_comp);
+    if (m_ictx->closing)
+      req->set_op_flags(LIBRADOS_OP_FLAG_FADVISE_DONTNEED);
```

Comment:
In the case of live migration, you wouldn't necessarily force-evict pages from the cache via DONTNEED.